typegate/tests/runtimes/python_wasi/python_wasi.ts

```diff
@@ -10,9 +10,23 @@ const tpe = t.struct({
   "b": t.list(t.either([t.integer(), t.string()])),
 });
 
-typegraph("python_wasi", (g: any) => {
+export const tg = await typegraph("python_wasi", async (g: any) => {
   const python = new PythonRuntime();
   const pub = Policy.public();
+  let identityModule;
+  try {
+    identityModule = await python.import(
+      t.struct({ input: tpe }),
+      tpe,
+      {
+        name: "identity",
+        module: "typegate/tests/runtimes/python_wasi/py/hello.py",
+        deps: ["typegate/tests/runtimes/python_wasi/py/nested/dep.py"],
```

Comment:
And it would be better if we had the paths relative to where the typegraph is...

Reply:
Ok, this was an issue because the cwd of the test and the tg_deploy was different. I will look into it once more after I finished with the other issues in hand.

Reply:
It's done.